### PR TITLE
resolve includes relative to proto root

### PR DESCRIPTION
**Pull request by hvoecking**

### CODE REVIEW

File: test/p.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
```
Comment (hvoecking) [inline test/p.json]: Check if we can avoid fixtures by checking for specific includes to be present

File: index.js [hunk 2]
```diff
@@ -284,41 +284,79 @@ var read = function(filename, options, schemas, cb) {
...
-    schemas.forEach(qualifyFieldTypes)
-    propagateExtends(schemas)
-    var sch = mergeSchemas(schemas)
-    cb(null, sch)
-  })
+  if (!Array.isArray(filenames)) {
+    filenames = [filenames];
+  }
+
+
```
Comment (hvoecking) [inline index.js]: remove extra newline 
:+1: https://github.com/mbrtargeting/resolve-protobuf-schema/commit/7e3df0109cacf28af90b385d2623e27c2d145fe1

File: index.js [hunk 3]
```diff
@@ -284,41 +284,79 @@ var read = function(filename, options, schemas, cb) {
...
-  var schemas = readSync(filename, options, {})
+function readAndMergeSync(filenames, options) {
+  options = normalizeOptions(filenames, options)
+
+  if (!Array.isArray(filenames)) {
+    filenames = [filenames];
+  }
+
+  var schemas = [];
+  filenames.map(function(filename) {
```
Comment (hvoecking) [inline index.js]: Use for each
:+1: https://github.com/mbrtargeting/resolve-protobuf-schema/commit/36a14a51eba981f73e674aaf33bbbb23068d2e45

File: test/p.json (hunk 1 above)
Comment (hvoecking) [inline test/p.json]: :+1: https://github.com/mbrtargeting/resolve-protobuf-schema/commit/1bf369e9fbe13bab4be540181fc6afa448608a25

**Comment by bratke** [comment]
LGTM

**Pull request merged**